modules/git/git.go

```diff
@@ -175,11 +222,18 @@ func InitOnceWithSync(ctx context.Context) (err error) {
 	}
 
 	initOnce.Do(func() {
-		err = InitSimple(ctx)
-		if err != nil {
+		if err = InitSimple(ctx); err != nil {
+			return
+		}
+		if err = initFixGitHome117rc(); err != nil {
 			return
 		}
 
+		// when git works with gnupg (commit signing), there should be a stable home for gnupg commands
+		if _, ok := os.LookupEnv("GNUPGHOME"); !ok {
+			_ = os.Setenv("GNUPGHOME", HomeDir())
+		}
```

Comment:
```suggestion
		if _, ok := os.LookupEnv("GNUPGHOME"); !ok {
			_ = os.Setenv("GNUPGHOME", filepath.Join(HomeDir(), ".gnupg"))
		}
```

Otherwise, the whole gpg folder content is at the same level with .gitconfig.
(Hopefully the suggestion is valid Go indentation. 😅)